src/Annotation/Traits/ActionTrait.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * @author Szyon Kunowski <szymon.kunowski@gmail.com>
+ */
+
+namespace Opstalent\CrudBundle\Annotation\Traits;
+
+use Doctrine\Common\Annotations\AnnotationException;
+use Opstalent\CrudBundle\Annotation\AbstractAnnotation;
+
+/**
+ * Trait ActionTrait
+ * @package Opstalent\CrudBundle
+ */
+trait ActionTrait
```

Comment:
why this trait is not an abstract class which would be base for our annotations?